ui/inspectors/tunable.reel/tunable.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @module ui/tunable.reel
+ */
+var Component = require("montage/ui/component").Component,
+    Model = require("core/model/model").Model,
+    TunableType = require("core/model/enumerations/tunable-type").TunableType;
+
+/**
+ * @class Tunable
+ * @extends Component
+ */
+exports.Tunable = Component.specialize(/** @lends Tunable# */ {
+    typeOptions: {
+        value: null
+    },
+
+    enterDocument: {
+        value: function(isFirstTime) {
+            var self = this;
+            if (isFirstTime) {
+                this.typeOptions = TunableType.members;
```

Comment:
This should be moved in the `templateDidLoad` method to ensure there is no flickering due to the options loading.